src/Resources/views/storefront/page/search/search-pagelet.html.twig

```diff
@@ -16,7 +16,8 @@
         {% if page.header.extensions.flConfig.filterPosition is constant('\\FINDOLOGIC\\FinSearch\\Findologic\\FilterPosition::LEFT') %}
             {% sw_include '@FinSearch/storefront/page/search/layout/leftFilterLayout.html.twig' %}
         {% else %}
-            {% sw_include '@FinSearch/storefront/page/search/layout/topFilterLayout.html.twig' %}
+            {# The top layout is the Shopware default #}
+            {{ parent() }}
```

Comment:
Since we call parent above as well, we could check for `FilterPosition::TOP` above and remove this parent call.

Reply:
This way, we would not set the `dataUrl` and `filterUrl` variables